community/cypher/compatibility-suite/src/main/java/cypher/feature/parser/generated/FeatureResultsBaseListener.java

```diff
@@ -17,7 +17,7 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-// Generated from /Users/mats/gitRoots/3.0-copy/neo4j/community/cypher/compatibility-suite/src/main/resources/FeatureResults.g4 by ANTLR 4.5.1
+// Generated from /Users/mats/gitRoots/3.0-main/neo4j/community/cypher/compatibility-suite/src/main/resources/FeatureResults.g4 by ANTLR 4.5.1
```

Comment:
Right. And I think this grammar will become stable soon enough to make it not so bad to just keep this line manually removed. But not just yet.